pylint/config/callback_actions.py

```diff
@@ -52,6 +52,27 @@ def __call__(
         return None
 
 
+class _ExtendAction(argparse._AppendAction):
+    """Action that adds the value to a pre-existing list.
+
+    It is directly copied from the stdlib implementation which is only available
+    on 3.8+.
```

Comment:
Maybe we could start a TODO list for when we drop 3.7 like #4683 to not forget to cleanup that kind of thing.

Reply:
I think whenever we drop support for a version we're likely to do a `grep` for `3.8` `3, 8` and other combinations. But feel free to open one!

Reply:
https://github.com/PyCQA/pylint/issues/6306